tests/core/error/valid/extra-content.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../../../core/error.rng"?>
+
+<problem xmlns="urn:ietf:rfc:7807" xmlns:test="http://libero.pub/test">
+
+    <status>404</status>
+    <test:foo test:bar="baz">qux <test:quxx test:quuz="corge"/></test:foo>
```

Comment:
This is key to allow detailed information to be returned, e.g. a validation error